docs/building.md

```diff
@@ -31,6 +31,41 @@ make all
 to generate the contract as well as all the example DAOs' initial storage
 defined below.
 
+## Generating a contract metadata
+
+BaseDAO metadata should be compliant with [TZIP-16](https://gitlab.com/tzip/tzip/-/blob/master/proposals/tzip-16/tzip-16.md).
+
+
+To print the metadata you'll need the `stack` tool, see
+[The Haskell Tool Stack](https://docs.haskellstack.org/en/stable/README/) tutorial
+for instructions on how to obtain it.
+
+```bash
+make metadata > metadata.json
+```
+
```

Comment:
```suggestion

You can then modify the produced `metadata.json` if you want to add more information.

```
I remembered now that we previously had a similar explanation as we only produce **known** parts of the metadata (hence the previous comment as well.

Reply:
I've push an update on this, as well as handle other changes. Have to update makefile a bit since its output was not as expected.